samples/tab-deeplink/nodejs/README.md

```diff
@@ -125,6 +125,17 @@ Enter text in the emulator.  The text will be echoed back by the bot.
 
 ![Meeting stage view](Images/MeetingStageView.png)
 
+6. **Deeplink to meeting side panel:**
+
+`@mention` bot in meeting chat to get adaptive card.
+
+![Bot-Adaptive-Card](DeepLinkBot/images/BotAdaptiveCard.png)
+
+Click on `Side Panel Deeplink` which will redirect to meeting side panel.
```

Comment:
If suppose meeting is not going on, what will happen?

Kindly add that flow as well.